### PR TITLE
card swiping and new design

### PR DESCRIPTION
**Pull request by kalidiagne**
- [x] Enable jubmoji card swiping navigation 
- [x] Update Jubmoji card design 
- [x] Update Onboarding design and card sizes 
- [x] add animation on card swiping
- [x] disable nav swipe navigation

### CODE REVIEW

File: .idea/.gitignore
Comment (AndrewCLu) [inline .idea/.gitignore:1]: Can we remove these .idea files?

**Review by AndrewCLu — COMMENTED** [review]
Thanks for this! A few comments:
1. Can we add a little animation when swiping between Jubmojis?
2. Images take a while to load, so when I swipe to a new Jubmoji it shows me a grey card for a while. Is there a quick way to implement prefetching images? If not, we can save for a future ticket, but I think this does make the experience a little more difficult.
3. Whichever Jubmoji I am currently looking at should be centered in the card view below, it should be in the middle of the carousel
4. The swiping across the carousel at the bottom doesn't work very well, right now its hard to control how much I am swiping. I think we should probably just remove swiping across the bottom for now

File: .idea/.gitignore
Comment (kalidiagne) [reply .idea/.gitignore:1]: Images are super heavy, the first thing we can also do on our side is to decrease the dimension (+2MB for the image is a lot, I think we can drastically reduce the size) 
Will work on the rest and update the PR

**Comment by vb7401** [comment]
Amazing work so far! Few design comments from testing:

- Peapod / first image in scroll
- Selected emoji on bottom doesn't change the top image
- More info is hidden behind the bottom scroll right now
- Swipe works for a while and then it stops changing from the first item

IMG_6941
IMG_6940

**Pull request closed without merging**